.github/workflows/lint-translations.yml

```diff
@@ -0,0 +1,58 @@
+name: lint-translations
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'i18n/*'
+
+jobs:
+  diff:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.head_ref }}
+        fetch-depth: 0
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+
+    - name: Install Dependencies
+      run: yarn add i18n-ai-translate
```

Comment:
nit: If we install globally instead, it won't try to check for intercompatibility with our existing modules, which should save a lot of time